### PR TITLE
Docs: Convert "getting started - zen of wagtail" from RST to MD

### PR DESCRIPTION
**Pull request by tbrlpld**
Partially replaces #6834

Diff locally with 
```
git diff master:./docs/getting_started/the_zen_of_wagtail.rst ./docs/getting_started/the_zen_of_wagtail.md
```

### CODE REVIEW

File: docs/getting_started/the_zen_of_wagtail.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# The Zen of Wagtail
+
+Wagtail has been born out of many years of experience building websites, learning approaches that work and ones that don\'t, and striking a balance between power and simplicity, structure and flexibility. We hope you\'ll find that Wagtail is in that sweet spot. However, as a piece of software, Wagtail can only take that mission so far - it\'s now up to you to create a site that\'s beautiful and a joy to work with. So, while it\'s tempting to rush ahead and start building, it\'s worth taking a moment to understand the design principles that Wagtail is built on.
```
Comment (kaedroho) [inline docs/getting_started/the_zen_of_wagtail.md:3]: Not sure why we need to escape quotes in markdown?

Comment (tbrlpld) [reply docs/getting_started/the_zen_of_wagtail.md:3]: I don't think we do. This is just how it comes out of Pandoc... I can remove those.

Comment (kaedroho) [reply docs/getting_started/the_zen_of_wagtail.md:3]: Thanks, this PR looks good to me otherwise!

Comment (tbrlpld) [reply docs/getting_started/the_zen_of_wagtail.md]: Fixed in eb1a17cfa62e1bd34b5b6535ea09b4483f65e5ef

**Review by kaedroho — APPROVED** [review]

**Comment by kaedroho** [comment]
Thanks!

**Pull request merged**